tests/numpy/product1.py

```diff
@@ -0,0 +1,6 @@
+import numpy as np
```

Comment:
We cannot accept manual tests anymore. Every new unit test should be run automatically by Pytest.